docs/helpers/Puppeteer.md

```diff
@@ -1879,11 +1879,11 @@ I.waitForRequest(request => request.url() === 'http://example.com' && request.me
 
 ### waitForResponse
 
-Waits for a network request.
+Waits for a network response.
```

Comment:
Related #2632

Reply:
oh, if that PR is merged, why do we have them here again, thou?

Reply:
Hmm...
I ran `npm run docs` command, and then this changes occured.
So, I suspend that PR ( #2632 ) was merged but the command, i.e. `npm run docs` , was not ran.

If this changes is not unnecessary,   should I delete this changes ?